gobblin-data-management/src/main/java/gobblin/data/management/conversion/hive/HiveSource.java

```diff
@@ -126,24 +139,27 @@
 
         // Create workunits for partitions
         if (HiveUtils.isPartitioned(hiveDataset.getTable())) {
-          List<Partition> sourcePartitions = HiveUtils.getPartitions(hiveDataset.getClientPool().getClient().get(),
-              hiveDataset.getTable(), Optional.<String> absent());
+          List<Partition> sourcePartitions =
+              HiveUtils.getPartitions(hiveDataset.getClientPool().getClient().get(), hiveDataset.getTable(),
+                  Optional.<String> absent());
 
           for (Partition sourcePartition : sourcePartitions) {
             LongWatermark lowWatermark = watermaker.getPreviousHighWatermark(sourcePartition);
+            if (globalLowWatermark.isPresent()) {
+              lowWatermark = globalLowWatermark.get();
```

Comment:
Yes this is a global setting across all datasets. Dataset level low watermark will be added with config store support.